py/selenium/webdriver/firefox/firefox_binary.py

```diff
@@ -150,7 +150,10 @@ def _get_firefox_start_cmd(self):
         """Return the command to start firefox."""
         start_cmd = ""
         if self.platform == "darwin":  # small darwin due to lower() in self.platform
-            start_cmd = "/Applications/Firefox.app/Contents/MacOS/firefox-bin"
+            start_cmd = self.which(ffname)
```

Comment:
is `ffname` in scope here?

Reply:
interesting, from what I can find too `iceweasel` is not supported on `osx` too; one for another ticket tho - Thanks